lib/repl.js

```diff
@@ -1546,10 +1547,14 @@ function complete(line, callback) {
     if (completionGroups.length && filter) {
       const newCompletionGroups = [];
       ArrayPrototypeForEach(completionGroups, (group) => {
-        const filteredGroup = ArrayPrototypeFilter(
-          group,
-          (str) => StringPrototypeStartsWith(str, filter)
-        );
+        const filteredGroup = ArrayPrototypeFilter(group, (str) => {
+          // Filter is always case-insensitive following chromium autocomplete
+          // behavior
+          return StringPrototypeStartsWith(
+            StringPrototypeToLocaleLowerCase(str),
+            StringPrototypeToLocaleLowerCase(filter)
```

Comment:
We could cache the value of `StringPrototypeToLocaleLowerCase(filter)` to avoid computing it at each iteration.

Reply:
Fixed